src/firefly/js/templates/lightcurve/LcViewer.jsx

```diff
@@ -105,6 +105,12 @@ export class LcViewer extends PureComponent {
                     errorMsg = `The uploaded table is not valid. The ${getMissionName(converterId)} option requires frame_id, source_id, or both scan_id and frame_num.
                     Please select the "Other" upload option for tables that do not meet these requirements.`;
                 }
+
+                if (converterId === 'ptf') {
```

Comment:
`error` is a string and meant to be used as an error message.  So, the code here can simply display the error message if exists.  It should not even care about converterId.  Mission specific message can be set in `onNewRawTable`.  You can see `wiseOnNewRawTable` as an example.